Infrastructure/Utils/PlayerValidation.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentValidation;
+using Infrastructure.DTOs;
+
+
+public class PlayerValidation : AbstractValidator<PlayerDTO>
```

Comment:
It's possible to validate an entity with attributes, it's easier to read and place entity validation logic together with entity declaration.

Reply:
added attribute validation